emitter/src/storage.rs

```diff
@@ -142,3 +144,33 @@ pub fn set_drop_status(e: &Env, new_status: bool) {
         LEDGER_BUMP_SHARED,
     );
 }
+
+/// Get the last block an emission fork was executed
+///
+/// Returns true if the emitter has dropped
+pub fn get_last_fork(e: &Env) -> u32 {
+    e.storage().persistent().bump(
+        &EmitterDataKey::DropStatus,
+        LEDGER_THRESHOLD_SHARED,
+        LEDGER_BUMP_SHARED,
+    );
+    e.storage()
+        .persistent()
+        .get(&EmitterDataKey::LastFork)
+        .unwrap_optimized()
```

Comment:
we should probably store this in the instance

Reply:
it's going to basically always be expired if it's persistent

Reply:
done

Also made drop_status an instance as it will also almost always be expired